Dockerfile

```diff
@@ -30,10 +30,6 @@ RUN go mod download
 # Copy the sources
 COPY ./ ./
 
```

Comment:
Why do you want to delete this?

Reply:
Okay read your comment. Some important notes:
* this is not the final image which we realease
* it's absolutelty essential for development: https://cluster-api.sigs.k8s.io/developer/tilt.html

Everyone else has it. I would really like to keep it